gateway-service/src/main/java/org/zowe/apiml/gateway/security/service/zosmf/AbstractZosmfService.java

```diff
@@ -149,14 +149,12 @@ protected String getURI(String zosmf) {
     protected RuntimeException handleExceptionOnCall(String url, RuntimeException re) {
         if (re instanceof ResourceAccessException) {
             if (re.getCause() instanceof SSLHandshakeException) {
-                log.error("SSL Misconfiguration, z/OSMF is not accessible. Please verify the following: \n" +
-                " - CN (Common Name) and z/OSMF hostname have to match.\n" +
-                " - Certificate is expired\n" +
-                " - TLS version match\n" +
-                "Further details and a stack trace will follow", re);
+                apimlLog.log("org.zowe.apiml.security.auth.zosmf.sslError");
+            } else {
+                apimlLog.log("org.zowe.apiml.security.serviceUnavailable", url, re.getMessage());
             }
-            apimlLog.log("org.zowe.apiml.security.serviceUnavailable", url, re.getMessage());
-            return new ServiceNotAccessibleException("Could not get an access to z/OSMF service.");
+            log.debug("ResourceAccessException accessing {}", url, re);
+            return new ServiceNotAccessibleException("Could not get an access to z/OSMF service.", re);
```

Comment:
I would just show the exception message instead of the full stack trace.

Reply:
I think in debugging the stack trace can be useful, especially if there are chained causes. We should not show stack traces if debugging is not enabled but once we do we want to have as much information as possible.